client/src/api.ts

```diff
@@ -625,14 +627,31 @@ export async function uploadVideo(
         question,
         uploadStatus: UploadStatus.PENDING,
         uploadProgress: (parseInt(progressEvent.loaded) / video.size) * 100,
+        tokenSource: tokenSource,
+        taskId: "",
       }),
     headers: {
       "Content-Type": "multipart/form-data",
     },
+    cancelToken: tokenSource.token,
   });
   return result.data.data;
 }
 
+export async function cancelUploadVideo(
+  mentorId: string,
+  question: Question,
+  taskId: string
+): Promise<CancelJob> {
+  const data = new FormData();
+  data.append(
+    "body",
+    JSON.stringify({ mentor: mentorId, question: question._id, task: taskId })
+  );
+  const result = await uploadRequest.post("/answer/cancel", data);
```

Comment:
I think this should just be:

```typescript
const result = await uploadRequest.post(
  "/answer/cancel", 
  { mentor: mentorId, question: question._id, task: taskId }
);
```

...or if not, why not?

Reply:
it is